### ISSUE TITLE
SVG image boundaries don't recognise line thickness in high quality pen mode

### ISSUE THREAD

**Issue by DroneBetter** [TurboWarp/scratch-vm#62]
When a sprite is an SVG, the rectangular footprint rendered is determined by the outermost part of a shape's outline, not that of the line following it. This leads to the edges being cropped off, as seen on the bottom and right edges of pic related. This is circumvented by drawing a transparent rectangle around the actual desired boundary, not an ideal solution.
<img width="115" alt="image" src="https://user-images.githubusercontent.com/58664547/128267936-625f1b72-f3a1-4a58-9d04-5592a58c0646.png">

**Comment by GarboMuffin** [comment]
How does the costume look when not stamped (as in, just when used as a sprite costume)? I think this is just a Scratch bug

**Comment by poly2it** [comment]
It still renders with the SVG issues when not stamped.

**Comment by supervoidcoder** [comment]
I know this is old, but If I understand this correctly, this is not Turbowarp's fault or anyone contributing to it, rather just a Scratch bug that I have seen even in the regular editor. It has always happened to me, and the only ones that can fix it are the Scratch team.

**Comment by CST1229** [comment]
> I know this is old, but If I understand this correctly, this is not Turbowarp's fault or anyone contributing to it, rather just a Scratch bug that I have seen even in the regular editor. It has always happened to me, and the only ones that can fix it are the Scratch team.

Mods can definitely fix it.

**Comment by supervoidcoder** [comment]
> > I know this is old, but If I understand this correctly, this is not Turbowarp's fault or anyone contributing to it, rather just a Scratch bug that I have seen even in the regular editor. It has always happened to me, and the only ones that can fix it are the Scratch team.
> 
> Mods can definitely fix it.

Yeah, that's true, but it would only work for that mod, and you would have to go through the code of the Scratch Engine just to find a bug in it and change the process of rendering objects on screen as well. It would probably take too long, and the Scratch Team is the only one that can universally change it without the need of a mod.

**Comment by GarboMuffin** [comment]
To be clear, as a mod we have full control over every line of code and could just fix it the same way the Scratch Team could. We already fix several Scratch bugs. (also, speculation by random people does not help anyone)

The reason we probably won't fix this anytime soon is that it's really the Scratch Team's responsibility to make SVGs work good, and fixing this could conceivably break a couple projects that indirectly rely on this.

**Comment by supervoidcoder** [comment]
Exactly, that's kind of what I tried to say, I just meant that this is a bug in Scratch itself, so this shouldn't have been reported to the turbowarp issues as it is not the modder's fault. I think that you already said that if the bug is also in scratch, it shouldn't be said here.
